openlibrary/api.py

```diff
@@ -112,10 +112,11 @@ def get(self, key, v=None):
         response = self._request(key + '.json', params={'v': v} if v else {})
         return unmarshal(response.json())
 
-    def get_many(self, keys):
-        """Get multiple documents in a single request as a dictionary."""
-        if len(keys) > 500:
-            # get in chunks of 500 to avoid crossing the URL length limit.
+    def get_many(self, keys, max_length = 100):
```

Comment:
```suggestion
    def get_many(self, keys, max_length=500):
```
The default max length should be the original value, `500`, for the sake of backwards compatibility.  Other consumers of `get_many` may require a max length of 500 to operate properly.